docs/source/python/driver_manager.rst

```diff
@@ -21,15 +21,11 @@
 Driver Manager
 ==============
 
-The driver manager is a library that provides bindings to the ADBC C
-API.  It delegates to dynamically-loaded drivers.  This allows
-applications to use multiple drivers simultaneously, and decouple
-themselves from the specific driver.
+The ``adbc_driver_manager`` package provides a :term:`driver manager` interface
+to Python. The package provides two APIs:
```

Comment:
```suggestion
for Python. The package provides two APIs:
```

Reply:
ha! I had for at first and then swapped it to "to".